cypress/integration/collection/newsletter_subscribe.spec.ts

```diff
@@ -0,0 +1,58 @@
+describe('subscribe to newsletter', () => {
+   const user = cy;
+   beforeEach(() => {
+      user.visit('/collections/phone');
+   });
+
+   it('requires an email', () => {
+      user.findByRole('button', { name: /subscribe$/i }).click();
+      user
+         .findByRole('button', { name: /subscribe$/i })
+         .parent('form')
+         .contains(/insert a valid email/i);
+   });
+
+   it('prevents invalid email', () => {
+      user.findByPlaceholderText(/enter your email/i).type('test@example');
```

Comment:
Where is findByPlaceholderText defined? I don't see it in the cypress docs.

Reply:
@mlahargou I'm using [Testing Library](https://testing-library.com/docs/cypress-testing-library/intro) which adds a few useful utils for testing, and helps create tests that reads more like what the user is doing.

[Here](https://github.com/iFixit/react-commerce/pull/23/files#diff-807c668e3005f98d28ed29b2f43426bbc52ccaba3e5e113f8a4451244b4cf3f8R26) you can see where this is setup.